bson/src/main/org/bson/codecs/pojo/PojoBuilderHelper.java

```diff
@@ -16,22 +16,26 @@
 
 package org.bson.codecs.pojo;
 
+import org.bson.codecs.configuration.CodecConfigurationException;
+
+import java.beans.Introspector;
```

Comment:
I know I mentioned this class to you, but on further reflection (no pun intended) let's avoid this dependency on java.beans.Introspector.  In Java 9 this will force applications to pull in the jdk.desktop module, which is going to limit our deployment opportunities.  

http://download.java.net/java/jdk9/docs/api/java/beans/Introspector.html#decapitalize-java.lang.String-